app/javascript/components/enrollment/steps/Exposure.js

```diff
@@ -40,11 +40,18 @@ class Exposure extends React.Component {
       if (jurisdiction_id) {
         value = jurisdiction_id;
         axios.defaults.headers.common['X-CSRF-Token'] = this.props.authenticity_token;
-        axios.get(window.BASE_PATH + `/jurisdictions/${jurisdiction_id}/assigned_users/immediate`).then(response => {
-          if (response?.data?.assignedUsers) {
-            this.setState({ assignedUsers: response.data.assignedUsers });
-          }
-        });
+        axios
+          .get('/jurisdictions/assigned_users', {
+            params: {
+              jurisdiction_id,
+              scope: 'exact',
+            },
+          })
+          .then(response => {
```

Comment:
> As far as UI feedback...
> 
> 1. I think I typically see the option for the how many entries to show to the left of the pagination options. Might make sense to move it down there and have it be less wide to declutter the main options at the top of the table.
> 2. I also think that the search bar (and jurisdiction input for that matter) don't need to be that wide. If you move the entries input next to the pagination, and set the widths of the jurisdictions input, assigned users input, and search bar, I think that could all be arranged one one line pretty neatly.
> 3. A pattern I usually see when you have selectable rows (and that I've employed currently in the admin table changes I'm making) is that the actions don't appear until you have rows selected. You could do something similar here rather than having an unclickable button. Either way, I think the checkboxes should still be the first column of the table, with the Action button on the left as it currently is on production.
> 4. The `datalist` tag used for the jurisdiction dropdown can cause a bit of confusion I think. Since it's a dropdown that allows me to type in my own input, it's not clear that those are the only options to me since they almost look like autocomplete options, and so when I type in something that is not in those options, nothing happens. I would expect when I type in a jurisdiction that's not an option for it to say "Please enter valid option" or for there to be no rows showing, but that's not the case. You can type in "USA" in the demo data and see what I mean.

Thanks for the feedback!

1. Yeah, I think this kind of makes sense. I'm not sure how often users try to change the number of entries they can view (and it might be different now since they can see 25 instead of 15).

2. So the jurisdictions input kind of does need to be this wide because the jurisdiction paths in production are much longer than the ones that we see in dev/test. Here is an example of one of the longer ones: https://gitlab.mitre.org/2019-ncov/jurisdiction-resources/-/blob/master/jurisdictions.yml#L393 The full path of this would be: 'USA, Maine, Healthcare, Central Maine Health, Central Maine Medical Center', which takes up a lot more real estate than 'USA, State 1, County 1'. I do wish as well that all of this could fit onto one line as well and would agree that it would make it look cleaner.

3. So I think I could probably hide the action button rather than making it disabled when no users are selected. In terms of the styling, I personally thought it was more intuitive to have the checkboxes and actions to the right, but ultimately, I think this is up to Kelly and the users, so we can ask her for feedback. The general layout of these filters/inputs are somewhat based on where they currently are in the current version of the app with the exception of the 'Actions' button, which I think Adam kind of wanted to style better in terms of positioning.

4. I agree that the datalist option is not the most ideal type of input for the same reason that you mentioned. Sarah is actually using a new type of input from react-select for the language options in enrollment which we might be able to use for the jurisdiction inputs as well. However, the datalist implementation right now is meant to be consistent with all the other jurisdiction inputs in other parts of the app as well (patient page and enrollment), so the users are not foreign to the way this input would work. I think, though, there is definitely room for improvement here though, but maybe to be done in a later ticket where all of the jurisdiction inputs are updated at once.

@holmesie any thoughts?

Thanks again @kylietmo for all the thoughtful feedback!

Reply:
addressed here: https://github.com/SaraAlert/SaraAlert/commit/87c321ec4a3f68004540e6358fe9287feb921e21#diff-f47ee7968d66302708c5ecc8dd05e1dcR50